Graphs/Table/index.js

```diff
@@ -333,7 +333,7 @@ const TableGraph = (props) => {
         return (
             <div>
                 {label}
-                {stateSortOrder.column === dataKey && <SortIndicator sortDirection={stateSortOrder.order} />}
+                {stateSortOrder.column === dataKey && <SortIndicator sortDirection={stateSortOrder.order.toUpperCase()} />}
```

Comment:
this appears to be a round-about fix. instead, can we use 
```
sortDirection = sort.order === 'DESC' ? 'ASC' : 'DESC';
```
in handleScrollSorting()?
similar change in handleStaticSorting() as well

Reply:
@killanch 

We can not do this way because if we have been doing it this way then scroll sorting is not working.

Reply:
@nxanil  Scroll sorting will continue to work.
The problem will be with sorting without scroll. It will not work because lodash/orderBy used in handleStaticSorting() accepts only lowercase asc/desc.
I realized this only now, I am ok with current changes.